packages/ERTP/src/types-ambient.js

```diff
@@ -291,6 +291,14 @@
  * @property {() => Issuer<K>} getIssuer Gets the Issuer for this mint.
  * @property {(newAmount: Amount<K>) => Payment<K>} mintPayment
  * Creates a new Payment containing newly minted amount.
+ * @property {() => CopySet<Payment<K>>} getRecoverySet
```

Comment:
Whoever has the mint can get the recoverySet?  That seems to violate POLA.

Can this authority be separated from the rest of minting?

Reply:
I understand that it would be a major change to the Mint API to separate this, but bundling the authority to recall all outstanding payments with the authority to mint new payments seems excessive. People will naturally come to think of `Mint` primarily as the latter, and pass it to parts of their program that shouldn't have the former power. 

How does this mixing of authorities compare with the cost of another approach, or other considerations?

Reply:
> but bundling the authority to recall all outstanding payments

Just to be sure there is no misunderstanding on the "all": This is only all payments made my `mintPayments` that have not yet been deposited or burned. It does not include other outstanding payments, like those made by withdrawal from a purse that have not yet been deposited or burned. Those would only be in the recovery set of the purse they were withdrawn from.

Reply:
The scenario I'm imagining is that there might be multiple parts of a contract that might have a reason to call `mintPayment`. It seems like minting will be the top-of-mind affordance of a Mint, and so a contract writer might share the Mint directly with those multiple parts of the contract without consciously considering the extra authority it carries. It seems excessive that the authority to recall all the minted payments is bundled with each authority to mint new ones, 

Perhaps I'm being too picky, and we've already divided the authority up finely enough, but I can't tell..

Reply:
This is a good criticism! I'll think about it.

Reply:
As @Chris-Hibbert and I discussed, I got rid of a `getRecoverySet` method on mint, and instead added a new `mintRecoveryPurse` property to `IssuerKit`. This enables an IssuerKit creator to hold onto it, in order to recover freshly minted but not-yet deposited payments. Of course, code prior to this PR does not know about this new property, and so ignored it. But even in that case, because the `mintRecoveryPurse` is registered in baggage, it can be obtained by an emergency upgrade.

Done. Resolving.